nemo/collections/common/data/lhotse/cutset.py

```diff
@@ -522,6 +523,50 @@ def read_lhotse_as_conversation(config) -> tuple[CutSet, bool]:
     return cuts, is_tarred
 
 
+def sqa_cut_to_conversation(
+    cut: Cut, audio_locator_tag: str, token_equivalent_duration: float
+) -> NeMoMultimodalConversation:
+    """
+    Converts a lhotse Cut representing a SQA example into a NeMoMultimodalConversation.
+
+    The ``cut`` is expected to have ``question`` and ``answer`` fields.
+    """
+    if isinstance(cut, NeMoMultimodalConversation):
+        return cut
+    turns = [
+        TextTurn(value=cut.question, role="user"),
+        AudioTurn(cut=cut, role="user", audio_locator_tag=audio_locator_tag, text=getattr(cut, "text", "")),
```

Comment:
the text would be typically under `cut.supervisions[0].text` (but we don't use it anyway, so it's fine)